README.md

```diff
@@ -201,6 +184,8 @@ Don't forget to give the project a star! Thanks again!
 4. Push to the Branch (`git push origin feature/AmazingFeature`)
 5. Open a Pull Request
 
+See the [open issues](https://github.com/dawnwages/wagtail-indymeet/issues) for a full list of proposed features (and known issues).
+
```

Comment:
At the top there is a "Request Feature" and "Report Bug" which should point to the issues area (#186), I'm not sure this is needed?

Reply:
Ah OK, none of the three links at the top point to anything. Shall I delete the top ones or this? I don't mind 🤔

Reply:
#186 will populate those links.